src/pages/IndexPage.vue

```diff
@@ -4,7 +4,12 @@
       <div class="q-mb-xl">
         <q-input v-model="tempData.name" label="姓名" />
         <q-input v-model="tempData.age" label="年齡" />
-        <q-btn color="primary" class="q-mt-md">新增</q-btn>
+        <q-btn
+          color="primary"
+          class="q-mt-md"
+          @click="addData(tempData.name, tempData.age)"
```

Comment:
這裡的 addData 永遠的是傳入 tempData ，可以不用填入參數，由 addData 去取得就好

```suggestion
          @click="addData"
```